# ----------------
# door_config.yaml
# ----------------
door:
  ejepuerta:
    x: -0.5
    y: 0.4
    z: 0.7
    orientacion: 0.57 # radianes
  apertura:
    picaporte:
      giro: -0.5236 # radianes
      numero_puntos: 10
    puerta:
      giro: -0.5236 # radianes
      numero_puntos: 10
